drivers/dma/dma_intel_adsp_gpdma.c

```diff
@@ -367,10 +399,11 @@ static const struct dma_driver_api intel_adsp_gpdma_driver_api = {
 		},							\
 	};								\
 									\
+	PM_DEVICE_DT_INST_DEFINE(inst, gpdma_pm_action);		\
```

Comment:
Good catch 😺 
I replaced PM_DEVICE_RUNTIME with PM_DEVICE. Now it will compile. If this is not the right way then I see only one solution, add dependencies.